frontend/src/app/interfaces/websocket.interface.ts

```diff
@@ -65,3 +66,11 @@ export interface IBackendInfo {
   gitCommit: string;
   version: string;
 }
+
+export interface Recommendedfees {
+  fastestFee: number;
+  halfHourFee: number;
+  hourFee: number;
+  minimumFee: number;
+  economyFee: number;
```

Comment:
I think the order should be preserved so @hunicus doesn't nitpick, but other than that looks good :)